operation/periodic-task/airflow/workflows.py

```
@@ -0,0 +1,128 @@
+    
+    1. Create database table with SQL DDL using `CREATE TABLE`.
+    2. Import CSV from HTTP location into CrateDB table using `COPY FROM`.
+    3. Export table from CrateDB into S3 bucket using `COPY TO`.
+    """
+
+    params = {
+        "table": "testdrive.weather_data",
+    }
+
+    ddl = SQLExecuteQueryOperator(
+        task_id=f"submit_ddl",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)